page-objects/pages/TASKListPage.js

```diff
@@ -131,10 +131,10 @@ async function clickAtTask(taskName) {
 			}
 		}
 	}
-	if (clickOnThatTask.isDisplayedInViewport()) {
+	if (!(clickOnThatTask.isDisplayedInViewport())) {
```

Comment:
It looks like the function 'isDisplayedInViewport' is not declared anywhere.

Reply:
The problem is not like that. In the function we are using `let clickOnThatTask = (await getTaskFromTaskOverview(taskName));` and it either returns an object or boolean value i.e. false. If you see in the function. For the object it works but when due to some reason when it runs too fast in CI/CD it returns `false` than its gives this error. Because `isDisplayedViewport` function needs object. Thanks for mentioning it.